app/core/base.js

```diff
@@ -157,6 +157,10 @@ FauxtonAPI.setSession = function (newSession) {
   return FauxtonAPI.session.fetchUser();
 };
 
-FauxtonAPI.reducers = [];
+FauxtonAPI.reducers = {};
+
+FauxtonAPI.addReducers = (reducers) => {
+  FauxtonAPI.reducers = Object.assign({}, FauxtonAPI.reducers, reducers);
+};
 
```

Comment:
Can't we just add the npm package and it will work? I would be happy for you to do that in this PR.